Dockerfile

```diff
@@ -0,0 +1,22 @@
+# Basic docker image for Monocle
+# Usage:
+#   docker build -t Monocle
+#   docker run -d --name Monocle -P Monocle
+
+FROM python:3.6
```

Comment:
I also try with :alpine, but I had a mistake with the pogeo requirement.

This is my current working docker file :

```dockerfile
FROM python:3.6.0-slim

WORKDIR /usr/src/app

ENV DUMP_INIT_VERSION 1.2.0
ENV TERM=xterm

RUN buildDeps='build-essential ca-certificates curl' \
 && apt-get update \
 && DEBIAN_FRONTEND=noninteractive \ 
    apt-get install --no-install-recommends -y $buildDeps python3-cairo libmysqlclient-dev libgeos-dev supervisor \
 && curl -sLo /tmp/dumb-init.deb https://github.com/Yelp/dumb-init/releases/download/v"$DUMP_INIT_VERSION"/dumb-init_"$DUMP_INIT_VERSION"_amd64.deb \
 && dpkg -i /tmp/dumb-init.deb \
 && pip install --upgrade pip \
 && pip install --no-cache-dir pymysql

COPY requirements.txt /usr/src/app/
RUN pip install --no-cache-dir -r requirements.txt

COPY optional-requirements.txt /usr/src/app/
RUN pip install --no-cache-dir -r optional-requirements.txt

# Cleanup
RUN apt-get purge -y --auto-remove $buildDeps \
 && apt-get autoremove -y \
 && apt-get clean \
 && rm -rf /var/lib/apt/lists/* /tmp/* /var/tmp/*

COPY . /usr/src/app/

EXPOSE 5000 6000
ENTRYPOINT ["/usr/bin/dumb-init"]
CMD ["/usr/bin/supervisord", "-c", "supervisord.conf"]
```

And my `supervisord.conf` : 
```
[supervisord]
nodaemon=true

[program:scan]
command=python scan.py
startsecs=5
stdout_logfile=/dev/stdout
stdout_logfile_maxbytes=0
stderr_logfile=/dev/stderr
stderr_logfile_maxbytes=0

[program:web]
command=python web.py --host 0.0.0.0
startsecs=5
```

And my docker CMD : 
```shell
docker run -d \
    --restart=always \
    --name pogomap \
    -p 6000:6000 \
    -v $path/accounts.csv:/usr/src/app/accounts.csv \
    -v $path/config.py:/usr/src/app/monocle/config.py \
    -v $path/landmarks.pickle:/usr/src/app/pickles/landmarks.pickle \
    -v $path/supervisord.conf:/usr/src/app/supervisord.conf \
    --link maria_db \
    registry.gitlab. . ./monocle:prod
```